engfrosh_site/management/urls.py

```diff
@@ -7,12 +7,8 @@
     path('discord/magic-link', views.get_discord_link),
     path('users/add', views.bulk_register_users),
     path('teams', views.manage_frosh_teams, name="manage_frosh_teams"),
-    path('users/add-to-guild', views.add_discord_user_to_guild, name="add_users_to_discord_guild"),
```

Comment:
This isn't the correct url to remove, this one doesn't match up with the view you got rid of, and this page is currently, very useful.

Reply:
Just reverted that change, accidentally removed the wrong view